packages/cosmic-swingset/x/swingset/handler.go

```diff
@@ -119,18 +119,18 @@ func handleMsgDeliverInbound(ctx sdk.Context, keeper Keeper, msg MsgDeliverInbou
 	if err != nil {
 		return sdk.ErrInternal(err.Error()).Result()
 	}
-	fmt.Fprintln(os.Stderr, "About to call SwingSet")
+	// fmt.Fprintln(os.Stderr, "About to call SwingSet")
```

Comment:
I don't see much point in commenting out print statements like these. If there was more state being reported, and one might forget to include enough  if it, it would be more worthwhile. As it is, it would be fine to remove them entirely.

Reply:
Okay, removed.